index.html

```diff
@@ -259,10 +259,200 @@ <h1>Architecture</h1>
+			The simplest Introduction mechanism, Direct, simply provides a single URL of a target TD.  
+			This assertion results in different minimal requirements depending on
+			which Introduction mechanism is selected out of the several available.
+			For example, when Direct is used as the sole Introduction mechanism,
+			at a minimum a D-Client must be able to accept a single URL pointing at a TD.
+			Note, however, that this TD could still be describing a Directory.
+			Another option would be for a D-Client to implement, as its sole 
+			Introduction mechanism, an "automatic" Introduction mechanism such as 
+			DNS-SD, possibly in combination with mDNS. 
+			In this case the application 
+			code running in the D-Client
+			would not have to supply any additional information to invoke Discovery.
+		</li>
+		<li><span class="rfc2119-assertion" id="consumer-must-support-fetching">
+			A D-Client MUST support fetching a TD from least one URL provided as part of the Introduction process.</span>
+			In other words, not only must a D-Client accept URLs pointing at TDs, it must be
+			able to fetch that TD, for example by using a GET for an HTTP URL or by reading it from a file 
+			specified by way of a file URL.
+			It is permissible to split the implementation of a D-Client into two parts, a "configurator" that
+			fetches/reads a TD and sets up a configuration during installation to be used by a 
+			run-time "executor" that cannot itself read TDs.
+			In this case, however, the "executor" is not by itself a D-Client.
+		</li>
+		<li><span class="rfc2119-assertion" id="consumer-may-multiple-intro">
+			A D-Client MAY support multiple invocations of the same Introduction mechanism.</span>
+			If multiple invocations of a single Introduction mechanism is supported, 
+			or if multiple Introduction mechanisms are used, then the D-Client needs to
+			manage a set of Introduction results.  
+			Some Introduction mechanisms themselves return a set of results, e.g. CoRE-RD, DID, or DNS-SD. 
+			Some, such as scanning Bluetooth beacons, may intrinsically result in separate multiple "invocations".
+			However, this D-Client requirement is optional to support an important special case: 
+			it is permissible to have a simple
+			D-Client that only supports a single invocation of a Direct Introduction, 
+			directly specifying a single TD via a URL.
+		<li><span class="rfc2119-assertion" id="consumer-merge-intros">
+			A D-Client MUST be able to merge URLs resulting from 
+			multiple Introduction mechanisms, multiple results from a single Introduction mechanism,
+			and multiple Introduction invocations into a single set.</span>
+			Note that even if only one Introduction mechanism is supported multiple results might be 
+			produced.  This assertion states that the overall output of the Introduction phase is
+			a single set of URLs.  
+			The word "set" is used here in the mathematical sense: the results are unordered.
```

Comment:
Hmmm... true, duplicate removal is also implied by "set".  I think that is ok and what we want, but maybe it could be emphasized here.  I still think I want to explicitly state duplicate removal in an assertion and not depend on the mathematical definition of "set".

Reply:
added "unordered and unique"